utils/bigclown/bigclown-mqtt2influxdb/Makefile

```diff
@@ -0,0 +1,48 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bigclown-mqtt2influxdb
+PKG_VERSION:=1.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/bigclownlabs/bch-mqtt2influxdb/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=5be14132311e85215abbfd732fe6cd652522ea0a343ee8ba7abab3ec7578eb99
+PKG_LICENSE:=MIT
+PKG_BUILD_DIR:=$(BUILD_DIR)/bch-mqtt2influxdb-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../../lang/python/python3-package.mk
+
+define Package/$(PKG_NAME)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=BigClown
+  URL:=https://github.com/bigclownlabs/bch-mqtt2influxdb
+  TITLE:=BigCLown MQTT to Influxdb bridge
+  DEPENDS:=+python3-paho-mqtt +python3-yaml +python3-influxdb +python3-jsonpath-ng +python3-schema
+endef
+
+define Py3Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/init $(1)/etc/init.d/bigclown-mqtt2influxdb
+	$(INSTALL_DATA) ./files/config.yml $(1)/etc/bigclown-mqtt2influxdb.yml
+endef
+
+define Package/$(PKG_NAME)/conffiles
+/etc/bc-mqtt2influxdb.yml
+endef
+
+define Build/Compile
+	sed -i 's/@@VERSION@@/$(PKG_VERSION)/' "$(PKG_BUILD_DIR)/setup.py"
+	$(call Py3Build/Compile/Default)
+endef
+
+$(eval $(call Py3Package,$(PKG_NAME)))
```

Comment:
it's been my experience with Python packages, that if you add a `python-<package>` or `python3-<package>` variant, there will be people asking for the other variant;
it's one of the "curses" or being a Python package maintainer, and one of the main reasons why `Py3Package` & `PyPackage` constructs exist ; 
[ though, in a funny way, I could also argue that they were added to piss people off ]

my only remark/question here is: when people ask for the other variant, will you:
a. refuse to add support for the other variant ?
b. add the other variant ?

in case of a. , they may choose to add their own variant/copy
in case of b. , you could prefix this now to `$(eval $(call Py3Package,python3-$(PKG_NAME)))` and leave room for a potential `python-bigclown` variant;

your choice :)

Reply:
These are not libraries but programs. It makes less sense to have them for both versions. Bigclown developers seems to support Python 2 as well but considering that Python 2 is end of live this year it makes little to no sense to me. So I am choosing a.

On the other hand if someone wants to have Python 2 support I don't see why there should not be one if upstream project supports it. I would expect that in such case it might be just build time option instead of full fledged package.

Either way I kind of hate idea of `python3-bigclown-mqtt2influxdb`. It somehow implies to me that it is python library not program that is written in python.

Reply:
ack; fine from me; let's leave the Python3 thingi out for now

Reply:
especially, if you will maintain them